lib/incoming_mail.rb

```diff
@@ -0,0 +1,110 @@
+module IncomingMail
```

Comment:
I think we could have more subfolders within `app` .... even `app/modesls`...I like the model suggested in this blog post: http://blog.codeclimate.com/blog/2012/02/07/what-code-goes-in-the-lib-directory/

Basically, anything that would potentially be re-usable by another app could go in `lib`. But this parser, for example, feels very particular to this app. Perhaps we could add it to `app/models/` if we make this a class. This is what they do in the griddler demo (griddler is a lib for parsing incoming mail, too) https://github.com/thoughtbot/griddler-demo/blob/master/app/models/comment_via_email.rb